src/explore-education-statistics-admin/src/pages/release/data/ReleaseDataFilePage.tsx

```diff
@@ -161,6 +165,26 @@ const ReleaseDataFilePage = ({
 
                 {getReplacementPlanMessage()}
 
+                {replacementDataFile?.status === 'FAILED' && (
+                  <>
+                    <p>
```

Comment:
Would make this a `WarningMessage` to make it more obvious that the user needs to take action.